be/test/storage/vectorized/schema_change_test.cpp

```diff
@@ -247,16 +247,16 @@ TEST_F(SchemaChangeTest, convert_datetime_to_date) {
 
     tm time_tm;
     strptime(origin_val.c_str(), "%Y-%m-%d %H:%M:%S", &time_tm);
-    TimestampValue timestamp = TimestampValue::create(2021, 9, 28, 0, 0, 0);
-    int64_t value = timestamp.timestamp();
+    int64_t value = ((time_tm.tm_year + 1900) * 10000L + (time_tm.tm_mon + 1) * 100L + time_tm.tm_mday) * 1000000L +
```

Comment:
Why change the calculation outside? There is no necessity.

Reply:
> Why change the calculation outside? There is no necessity.

This case is convert `Datetime` to `Date`, `Datetime` is store as `int64_t` in our storage engine. I think create `TimestampValue` first and convert it to `int64_t` is a bit confusing because it looks like this case is convert `Timestamp` to `Date`

Reply:
I worry the algorithm is dispersed in the too places, it will be error-prone.